### PR TITLE
Jrny 55 general uplift and relabel discover in readme files

### PR DESCRIPTION
**Pull request by mattsciotto**
This PR is a general cleanup of the example projects where applicable:
1. "Ringtail" renamed to "Discover"
2. "FTI" renamed to "Nuix"
3. Namespaces changed to "Discover"
4. "API" changed to "ApiToolkit"
5. Libraries/frameworks updated
6. "Api key" references removed as they are no longer needed in V2

### CODE REVIEW

File: dotnet/src/Discover.APIToolkit/Client.cs [hunk 1]
```diff
@@ -29,7 +29,6 @@ private void Initialize()
 
             HttpClient = new HttpClient();
             HttpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", Configuration.Token);
-            HttpClient.DefaultRequestHeaders.Add("ApiKey", Configuration.ApiKey);
```
Comment (ddougl1) [inline dotnet/src/Discover.APIToolkit/Client.cs:32]: Why is this removed?

**Review by ddougl1 — APPROVED** [review]

File: dotnet/src/Discover.APIToolkit/Client.cs (hunk 1 above)
Comment (mattsciotto) [reply dotnet/src/Discover.APIToolkit/Client.cs:32]: The new ("V2") version of the auth/jwt token in Discover doesn't require the "key" component separately as it is baked into the token now.

Comment (ddougl1) [reply dotnet/src/Discover.APIToolkit/Client.cs:32]: cool, thanks for confirming
